.changeset/sour-cobras-share.md

```diff
@@ -0,0 +1,8 @@
+---
+"wrangler": patch
+---
+
+feat: autogenerated config from dash
+
+related discussion #1623
+resolves #1638
```

Comment:
Could you add a description here of what this PR does?

Reply:
I can go into a bit more detail for sure.

Reply:
I don't see any additional description in the changeset file. Did you miss this?
Also the headline of this changeset doesn't seemt to be aligned with the changes in the PR.
AFAICT this PR is mainly switching over to use the non-legacy API endpoints and pulling the default environment.
Can we change the changeset to reflect this?